src/odemis/driver/test/xt_client_test.py

```diff
@@ -1014,26 +1014,28 @@ def test_type_scanner_child(self):
             self.assertIsNot(type(child), xt_client.Scanner)
 
     def test_delta_pitch_VA(self):
-        current_value = self.scanner.deltaPitch.value
+        init_value = self.scanner.deltaPitch.value
         # Test if directly changing the value via the VA works. Not always will the entirety of the range be
         # allowed. Negative delta pitch is limited by the voltage it can apply. Therefore the max range and the 0
         # value is tested.
-        for test_delta_pitch in (0.0, self.scanner.deltaPitch.range[1]):
+        current_value = init_value
+        for test_delta_pitch in (0.0, 0.5e-6, self.scanner.deltaPitch.range[1]):
             self.scanner.deltaPitch.value = test_delta_pitch
-            self.assertEqual(test_delta_pitch, self.scanner.deltaPitch.value)
-            self.assertEqual(test_delta_pitch, self.microscope.get_delta_pitch() * 1e-6)
+            current_value += test_delta_pitch
```

Comment:
If the current value was 0.5e-6 and you add the self.scanner.deltaPitch.value[1] (so the largest value allowed), it will raise an error, won't it?

Reply:
That is correct, it would go at of the allowed range.